src/corelibs/U2Gui/src/ObjectViewModel.cpp

```diff
@@ -248,8 +253,8 @@ void GObjectView::buildStaticToolbar(QToolBar *tb) {
     emit si_buildStaticToolbar(this, tb);
 }
 
-void GObjectView::buildStaticMenu(QMenu *m) {
-    emit si_buildStaticMenu(this, m);
+void GObjectView::buildMenu(QMenu *m, const QString &type) {
```

Comment:
Could you rename this function as `emitBuildMenu`? Cuz now it looks like this function builds menu itself, but it just emits a signal

Reply:
Not really agree, but OK